go-controller/pkg/config/config.go

```diff
@@ -189,7 +189,7 @@ var (
 	}
 
 	ClusterManager = ClusterManagerConfig{
-		V4TransitSwitchSubnet: "168.254.0.0/16",
+		V4TransitSwitchSubnet: "172.20.0.0/16",
 		V6TransitSwitchSubnet: "fd97::/64",
```

Comment:
how about the v6 one? is that in the clear?

Reply:
That's within the fc00::/7 range, which is reserved for unique local addresses, so that's ok. https://datatracker.ietf.org/doc/html/rfc6890#section-2.2.3

Reply:
It doesn't seem to be in use elsewhere either: https://github.com/search?q=org%3Aopenshift%20fd97%3A%3A&type=code

Reply:
Save folks a click, from `rfc6890`:
```
| Address Block        | fc00::/7     |
| Name                  | Unique-Local |
```
Expanded `/7` IP block: `fc00::1 - fdff:ffff:ffff:ffff:ffff:ffff:ffff:fffe`
`fd97::/64` clearly falls within it.